### PR TITLE
fix: Update screenshots on Social Sign In guide for Facebook & Google

### PR DESCRIPTION
**Pull request by nadetastic**
#### Description of changes:
Updated screenshots for social sign in guide for Facebook and Google on the following pages:

JS/RN
- https://docs.amplify.aws/lib/auth/social/q/platform/js/
- https://docs.amplify.aws/lib/auth/social/q/platform/react-native/

Swift
- https://docs.amplify.aws/lib/auth/social/q/platform/ios/#catching-errors
- https://docs.amplify.aws/lib-v1/auth/social/q/platform/ios/

Android
- https://docs.amplify.aws/lib/auth/social/q/platform/android/
- https://docs.amplify.aws/lib-v1/auth/social/q/platform/android/
- https://docs.amplify.aws/sdk/auth/hosted-ui/q/platform/android/

Flutter
- https://docs.amplify.aws/lib/auth/social/q/platform/flutter/
- https://docs.amplify.aws/lib-v1/auth/social/q/platform/flutter/

#### Related GitHub issue #, if available:
https://github.com/aws-amplify/docs/issues/5481
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [x] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [x] JS
- [x] iOS
- [x] Android
- [x] Flutter
- [x] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [x] Are any files being deleted with this PR? If so, have the needed redirects been created?
	- Image assets were replaced/deleted

- [x] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Comment by abdallahshaban557** [comment]
FYI @salihgueler

**Review by tjleing — DISMISSED** [review]
Should we also delete the images `facebook7` through `9`?

File: public/images/cognitoHostedUI/facebook4.png
Comment (tjleing) [inline public/images/cognitoHostedUI/facebook4.png:1]: In the previous photo `facebook6` app domains were non-empty, do we need to put the user pool domain here?

File: src/fragments/lib/auth/common/social_signin_web_ui/configure_auth_category.mdx [hunk 1]
```diff
@@ -1,54 +1,42 @@
...
-![Image](/images/cognitoHostedUI/facebook7.png)
-9. From the navigation bar choose *Facebook Login* and then *Settings*.
-10. Type your redirect URL into *Valid OAuth Redirect URIs*. It will consist of your user pool domain with the /oauth2/idpresponse endpoint.
+5. Under *Valid OAuth Redirect URIs* type your user pool domain with the `/oauth2/idpresponse` endpoint.
 
     `https://<your-user-pool-domain>/oauth2/idpresponse`
 
-    ![Image](/images/cognitoHostedUI/facebook8.png)
-11. Save changes.
+![Image](/images/cognitoHostedUI/facebook6.png)
```
Comment (tjleing) [inline src/fragments/lib/auth/common/social_signin_web_ui/configure_auth_category.mdx:21]: Picture says `your-cognito-endpoint` while text says `your-user-pool-domain`

**Review by israx — APPROVED** [review]
LGTM

**Review by salihgueler — APPROVED** [review]
If we are sure the pictures are good on different screensizes and resolutions, I am good as well.

**Review by abdallahshaban557 — APPROVED** [review]
LGTM

File: public/images/cognitoHostedUI/facebook4.png
Comment (nadetastic) [reply public/images/cognitoHostedUI/facebook4.png:1]: This isn't a requirement to get it working, and the app domain is the actual website url and not the userpool domain

File: src/fragments/lib/auth/common/social_signin_web_ui/configure_auth_category.mdx (hunk 1 above)
Comment (nadetastic) [reply src/fragments/lib/auth/common/social_signin_web_ui/configure_auth_category.mdx:21]: Fixing

**Review by harsh62 — APPROVED** [review]
Looked at the swift changed.. LGTM

**Review by jcjimenez — APPROVED** [review]

**Review by tjleing — APPROVED** [review]

**Review by hdworld11 — APPROVED** [review]

**Review by cshfang — APPROVED** [review]

**Review by katiegoines — APPROVED** [review]

**Pull request merged**